operator/validator/controller.go

```diff
@@ -686,6 +686,10 @@ func (c *controller) onShareStart(share *ssvtypes.SSVShare) (bool, error) {
 		c.logger.Warn("skipping validator until it becomes active", fields.PubKey(share.ValidatorPubKey))
 		return false, nil
 	}
+	if share.Invalid {
```

Comment:
maybe we can add it to the filter above in `setupValidators` like the `ByNotLiquidated`?